utils/resources.go

```diff
@@ -62,4 +62,14 @@ var ResourceRequirements = map[string]corev1.ResourceRequirements{
 			"memory": resource.MustParse("200Mi"),
 		},
 	},
+	"kube-rbac-proxy": {
+		Limits: corev1.ResourceList{
+			"memory": resource.MustParse("30Mi"),
+			"cpu":    resource.MustParse("50m"),
+		},
+		Requests: corev1.ResourceList{
+			"memory": resource.MustParse("30Mi"),
+			"cpu":    resource.MustParse("50m"),
+		},
+	},
```

Comment:
```yaml
        resources:
          limits:
            cpu: 500m
            memory: 128Mi
          requests:
            cpu: 5m
            memory: 64Mi
```
This is the current resource requirement specified by OSDK in `config/` for `kube-rbac-proxy` right now. Are we sure that our KRP usage will not exceed `30Mi` and `50m`?

Reply:
@rexagod  this is the configuration that we have in ocs-osd deployer, https://github.com/red-hat-storage/ocs-osd-deployer/blob/main/utils/resources.go#L211
I personally think limits in OSDK in config/are very high for a sidecar

Reply:
ACK!